tests/test_classifier/test_rocauc.py

```diff
@@ -213,10 +212,11 @@ def test_rocauc_quickmethod(self):
         Test the ROCAUC quick method
         """
         X, y = load_occupancy(return_dataset=True).to_numpy()
-        model = DecisionTreeClassifier()
+        model = LogisticRegression()
```

Comment:
At @pdamodaran's request I'm taking a quick look a this PR and I noticed this change - was just curious why it was required?

Reply:
@bbengfort - This change was not required, as the quick method works with all classifiers. I changed it to be consistent with most of the other tests in the module, but I can change it back if it causes confusion!

Reply:
Ok no worries - just wanted to check; usually when there is a change like this it was because the original model wasn't working for what seemed like a minor reason to the contributor but actually revealed a small bug in our code that we needed to fix!